lib/bgs/services/benefit_claims.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# As a work of the United States Government, this project is in the
+# public domain within the United States.
+#
+# Additionally, we waive copyright and related rights in the work
+# worldwide through the CC0 1.0 Universal public domain dedication.
+
+module BGS
+  class BenefitClaimWebServiceV1 < BGS::Base
+    def bean_name
+      'BenefitClaimWebServiceBean'
+    end
+
+    def self.service_name
+      'benefit_claims'
+    end
+
+    def find_claims_details_by_participant_id(participant_id:)
+      response = request(:find_bnft_claim_detail_by_ptcpnt_id, 'ptcpntId': participant_id)
+      response.body
```

Comment:
mostly for consistency with other services implemented in this gem, typically this will respond with the high level object, so `response.body[:find_bnft_claim_detail_by_ptcpnt_id_response]` here instead of `response.body`

Reply:
ah ok.
[Updated here](https://github.com/department-of-veterans-affairs/bgs-ext/pull/62/commits/404707deedd01285ef968fbc84487f6b0e465de6).